tests/python/unittest/test_gluon_rnn.py

```diff
@@ -278,7 +282,7 @@ def test_cell_fill_shape():
     cell = gluon.rnn.LSTMCell(10)
     cell.hybridize()
     check_rnn_forward(cell, mx.nd.ones((2, 3, 7)))
-    assert cell.i2h_weight.shape[1] == 7, cell.i2h_weight.shape[1]
+    assert cell.i2h.weight.shape[1] == 7, cell.i2h.weight.shape[1]
```

Comment:
This is actually a breaking change.

Also cell.params['i2h_weight'] wouldn't work.

Reply:
oh good catch!